src/components/dropdownMenu.js

```diff
@@ -0,0 +1,210 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react';
+import styled from '@emotion/styled';
+
+import { css } from 'theme-ui';
+import { InlineBlock } from './layout';
+import { Absolute, Relative } from './position';
+import { PropTypes } from 'prop-types';
+import {
+  applyVariation,
+} from './../utils/getStyles';
+import {
+  UP_ARROW,
+  DOWN_ARROW,
+  ENTER_KEY,
+  TAB_KEY,
+} from './../utils/general';
+
+const DropdownContainer = styled(InlineBlock)({
+  position: 'relative',
+});
+
+const DropDown = styled(InlineBlock)(
+  ({ theme, variant }) => applyVariation(theme, `${variant}.dropdownTrigger`, 'dropdownMenu')
+);
+
+const OptionsContainer = styled(Absolute)`
+  ${({ theme, positionTop }) => css({
+    width: '200px',
+    background: 'white',
+    py: 3,
+    borderStyle: 'solid',
+    borderWidth: '1px',
+    borderColor: 'borderGray',
+    borderRadius: 4,
+    bottom: positionTop,
+  })(theme)}
+  ${({theme, variant}) => applyVariation(theme, `${variant}.dropdownContainer`, 'dropdownMenu')}
+`;
+
+const Menu = styled(Relative)`
+  ${({ theme }) => css({
+    py: 2,
+    px: 3,
+    '&:hover': {
+      background: 'borderGray',
+    },
+  })(theme)}
+  ${({ theme, variant }) => applyVariation(theme, `${variant}.dropdownOption`, 'dropdownMenu')}
+  ${({ theme, variant, hover }) => hover ? applyVariation(theme, `${variant}.dropdownOption.&:hover`, 'dropdownMenu') : ''}
+  ${({ theme, variant, hover }) => hover ? applyVariation(theme, `${variant}.dropdownOption.&:focus`, 'dropdownMenu') : ''}
+`;
+
+const DropdownMenu = ({
```

Comment:
Is this a menu component or Input component?

Reply:
I guess you developed a menu component whereas what we need is an input.

Check the issue and the reference mentioned
https://github.com/Codebrahma/bricks/issues/14